core/system_chaincode/importsysccs.go

```diff
@@ -19,10 +19,18 @@ package system_chaincode
 import (
 	"github.com/hyperledger/fabric/core/system_chaincode/api"
 	//import system chain codes here
+	"github.com/hyperledger/fabric/bddtests/syschaincode/noop"
 )
 
 //see systemchaincode_test.go for an example using "sample_syscc"
-var systemChaincodes = []*api.SystemChaincode{}
+var systemChaincodes = []*api.SystemChaincode{
```

Comment:
@tamasblummer As you can see, deployed system chaincodes are determined at build time, again. (this is a change from master)